packages/grid/_modules_/grid/hooks/features/rows/useGridEditRows.ts

```diff
@@ -206,10 +267,33 @@ export function useGridEditRows(
       const rowUpdate = { ...row, [field]: value };
       apiRef.current.updateRows([rowUpdate]);
     },
-    [apiRef, logger],
+    [apiRef, logger, props.editMode],
+  );
+
+  const commitRowChange = React.useCallback(
+    (id: GridRowId, event?: React.SyntheticEvent): boolean => {
+      if (props.editMode === 'cell') {
+        throw new Error(`You can't commit changes when the edit mode is 'cell'`);
+      }
+
+      const model = apiRef.current.getEditRowsModel();
+      const row = model[id];
+      if (!row) {
+        throw new Error(`Row at id: ${id} is not being editted`);
+      }
+
+      const hasFieldWithError = Object.keys(row).some((field) => !!row[field].error);
```

Comment:
Could we use directly `Object.values` instead of extracting the keys and then getting the value of each key ?